cmd/debos/debos.go

```diff
@@ -122,6 +124,24 @@ func main() {
 		return
 	}
 
+
+	var chrootMethod debos.ChrootEnterMethod
+	switch options.ChrootMethod {
```

Comment:
code repetition here from https://github.com/go-debos/debos/blob/main/commands.go#L18; wonder if there's anything that can be done about that ? or maybe a function in that file e.g. `ChrootMethodFromString` ?